src/components/KSelect/KSelectItem.vue

```diff
@@ -14,6 +14,7 @@
         :disabled="item.disabled === true ? true : undefined"
         type="button"
         :value="item.value"
+        @click="handleClick"
```

Comment:
question: how was this working previously if this is required?

Reply:
It was previously bound to div on line 6. I moved it to button